doc/sphinx/source/quickstart/running.rst

```diff
@@ -18,20 +18,26 @@ An
 is available in the ESMValTool installation folder as
 ``examples/recipe_python.yml``.
 
-This recipe finds data from CanESM2 and MPI-ESM-LR for 2000 - 2002,
-extracts a single level (850 hPa), regrids it to a 1x1 degree mesh and runs
-a diagnostic script that creates some plots of Air temperature and
-precipitation flux. You can download the recipe from
-`github <https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/recipes/examples/recipe_python.yml>`_
-and save it in your project directory as (e.g.) ``recipe_python.yml``
-and then run ESMValTool with
+This recipe finds data from BCC-ESM1 and CanESM2 and creates two plot types:
+
+- a global map plot that shows the monthly mean 2m surface air temperature in
+  January 2000.
+- a time series plot that shows the globally averaged annual mean 2m surface
+  air temperature and compares it to the one in Amsterdam.
+
+You can download the recipe from `GitHub
```

Comment:
maybe mention how - click RAW and then copy paste etc - lots of people asked me how to download a file from GH when there is no Download button